macaw-loader-aarch32/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2018, Galois, Inc.
```

Comment:
Consider updating copyright year

Reply:
I'd also recommend a separate PR for this.  Although advice from our legal dept. is that churning these dates isn't necessary to maintain copyright.

Reply:
OK. For what it's worth, I've pushed a separate commit (https://github.com/GaloisInc/macaw/pull/470/commits/a0c7f7a4598a0327961f2561468514e5ebecd64a) that updates the copyright years, but only for the `macaw-loader` packages (since those are the ones that are actually touched in this PR). If you'd prefer, I could revert this and defer it to a more general PR.

Reply:
> Although advice from our legal dept. is that churning these dates isn't necessary to maintain copyright.

TIL, thanks! That should save us all some time and mundanity :grin:

Reply:
Not a big change and it's in a separate commit (thus bisectable), so don't worry about backing it out now.